utils/installer/install.sh

```diff
@@ -2,24 +2,37 @@
 set -eo pipefail
 
 #Set branch to master unless specified by the user
-declare -r LV_BRANCH="${LV_BRANCH:-rolling}"
-declare -r LV_REMOTE="${LV_REMOTE:-lunarvim/lunarvim.git}"
-declare -r INSTALL_PREFIX="${INSTALL_PREFIX:-"$HOME/.local"}"
+declare LV_BRANCH="${LV_BRANCH:-"rolling"}"
```

Comment:
Outer quotes are unnecessary for variables between braces
`LV_BRANCH=${LV_BRANCH:-"rolling"}`

Reply:
It was actually the other way around, but I thought you liked them to be consistent 😄